defaults/krill-pubd.conf

```diff
@@ -84,13 +84,24 @@
 #
 ### admin_token =
 
-# Specify the ip address and port number that the server will use.
+# Specify the ip addresses and port number that the server will use.
 #
-# Note: we recommend that you use the defaults and use a proxy if you
-# must make your Krill instance accessible remotely.
+# Note: by default Krill uses "127.0.0.1" (IPv4 localhost) as its IP address.
+# We recommend that you keep this setting and use a proxy server such as NGINX
+# or Apache if you must make your Krill instance accessible remotely.
 #
-### ip             = "localhost"
-### port           = 3000
+# You can use the 'ip' setting in this config file to override the default. You
+# can also specify multiple IP addresses if you need to. In this case 'ips' needs
+# to be set instead of 'ip'. If you set both, then 'ips' takes precedence.
+#
+# If you do change the IP address(es), then the service uri will use it (or the
+# first in case of multiple) as a hostname for remote CAs and Publishers. If you
+# want to use a proper public hostname, please set 'service_uri' below.
+#
+#
+### ip             = "127.0.0.1"            # default
+### ips            = [ "127.0.0.1", "::1" ] # overrides 'ip'
```

Comment:
FYI serde_as OneOrMany from the serde_with crate can be used to accept both single string or toml array of string inputs under the same toml key, avoiding the need for two options for the same thing.

Reply:
As most users won't need this I prefer that we do not introduce yet another dependency.

Reply:
alright.. `serde_with` only depends on `serde` and the rest is optional - so I guess that including it is low risk.. will look.